content/community/roadmap/studio/_index.nb.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Roadmap for Altinn Studio
+linktitle: Studio
+description: Høynivå roadmap og funskjonelle beskrivelser for Altinn Studio og funksjonalitet som understøtter apper utviklet der.
+weight: 900
+---
+
+{{% panel %}}
+For informasjon om **hvordan man bruker** funksjonalitetene beskrevet her, se seksjonene om [apper](/nb/app/) og [API](/nb/api).
```

Comment:
Pro tip: If you use relative links they will also work in github and in vscode preview, and can be the same across languages :)